src/block_cache/block_io.c

```diff
@@ -157,7 +189,9 @@ __wt_blkcache_read(WT_SESSION_IMPL *session, WT_ITEM *buf, const uint8_t *addr,
              * corrupted data, we'll end up here on corrupted data.
              */
             WT_ERR(__wt_scr_alloc(session, 0, &etmp));
-            if ((ret = __wt_decrypt(session, encryptor, WT_BLOCK_ENCRYPT_SKIP, ip, etmp)) != 0)
+            /* TODO: ENCRYPT_SKIP needs to skip the right size for an object header if it has one */
```

Comment:
@agorrod FYI I think this comment is outdated now that we have `bm->encrypt_skip`; I plan to remove it from the disagg block manager PR.